ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoController.m

```diff
@@ -40,18 +46,48 @@ - (void)viewDidLoad {
 
     MSFButton *testButton = [self createButtonWithTitle:@"Test" action:@selector(buttonPressed:)];
     [self.container addArrangedSubview:testButton];
+
+    [self setAddedLabels:[NSMutableSet set]];
+
+    self.appearanceController = [MSFDemoAppearanceControllerWrapper createDemoAppearanceControllerWithDelegate:nil];
+    [self configureAppearancePopover];
+
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector: @selector(themeDidChange:)
+                                                 name: @"FluentUI.stylesheet.theme"
+                                               object: nil];
 }
 
-- (void)buttonPressed:(id)sender {
+- (void)resetAddedLabels {
+    for (UILabel *label in [self addedLabels]) {
+        [label removeFromSuperview];
+    }
+    [[self addedLabels] removeAllObjects];
+}
+
+- (void)addLabelWithText:(NSString *)text
+               textColor:(UIColor *)textColor {
     MSFLabel *label = [[MSFLabel alloc] initWithStyle:MSFTextStyleHeadline colorStyle:MSFTextColorStyleRegular];
     [label setTextAlignment:NSTextAlignmentCenter];
-    [label setText:@"Test label with color"];
+    [label setText:text];
+    [label setTextColor:textColor];
+
+    [[self container] addArrangedSubview:label];
+    [[self addedLabels] addObject:label];
+}
 
+- (void)buttonPressed:(id)sender {
```

Comment:
Is there a reason we're adding labels on a button press? This seems like an unnecessary step.